src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size = 0;
+    private T[] arr;
```

Comment:
Avoid using non-informative variable names like 'arr'. Consider renaming it to something more descriptive, such as 'elements' or 'items', to reflect its purpose.